code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -145,6 +145,53 @@
 		GLOB.pre_setup_antags -= changeling
 	return TRUE
 
+//////////////////////////////////////////////
+//                                          //
+//              ELDRITCH CULT               //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/ecult
+	name = "Changelings"
+	antag_flag = ROLE_CHANGELING
```

Comment:
copy paste oh no

Reply:
oh shit didnt notice it thanks for pointing it out